org.eclipse.winery.repository.ui/src/app/wineryTableModule/wineryTable.component.ts

```diff
@@ -11,108 +11,16 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
  *******************************************************************************/
-import {Component, DoCheck, EventEmitter, Input, OnInit, Output} from '@angular/core';
-import {isNullOrUndefined} from 'util';
+import { Component, DoCheck, EventEmitter, Input, OnInit, Output, ViewChild } from '@angular/core';
+import { isNullOrUndefined } from 'util';
 
-/**
- * This component provides an easy and fast way to use the ng2-table with further modifications
```

Comment:
In my opinion, it distracts from the code and does not add real value. But i can also revert the change :)

Reply:
The documentation itself is good. I guess it will be okay to remove some of the available iput attributes, but creates a cool code documentation, if you run `npm run doc`

Reply:
oh i first thought its only docu in the code. I will just insert it again :)